code/game/machinery/suit_storage_unit.dm

```diff
@@ -553,15 +553,21 @@
 			dump_inventory_contents()
 
 /obj/machinery/suit_storage_unit/process(seconds_per_tick)
-	var/obj/item/stock_parts/cell/cell
-	if(suit && istype(suit))
-		cell = suit.cell
-	if(mod)
-		cell = mod.get_cell()
-	if(!cell || cell.charge == cell.maxcharge)
+	var/list/cells_to_charge = list()
+	for(var/obj/item/charging in list(mod, suit, helmet, mask, storage))
+		var/obj/item/stock_parts/cell/cell_charging = charging.get_cell()
+		if(!istype(cell_charging) || cell_charging.charge == cell_charging.maxcharge)
+			continue
+
+		cells_to_charge += cell_charging
+
+	var/cell_count = length(cells_to_charge)
+	if(cell_count <= 0)
 		return
 
-	charge_cell(final_charge_rate * seconds_per_tick, cell)
+	var/charge_per_item = final_charge_rate * seconds_per_tick / cell_count
```

Comment:
```suggestion
	var/charge_per_item = (final_charge_rate * seconds_per_tick) / cell_count
```

spooky pemdas monster i hate it it's evil

Reply:
even with PEMDAS this evaluates to the same thing, that's literally how math works. but i guess i'll do it anyways

Reply:
<Details>

<Summary>Cough</Summary>

![image](https://github.com/tgstation/tgstation/assets/51863163/f735b442-a1da-4c57-b4b2-4d201af36f0c)

</details>

Reply:
one day it's going to be mis-intrepreted and they're going to eat your liver with a ritz cracker